app/src/main/java/com/nmakademija/nmaakademija/adapter/ArticlesAdapter.java

```diff
@@ -64,4 +51,16 @@ public void onClick(View view) {
     public int getItemCount() {
         return articlesList.size();
     }
+
+    public class MyViewHolder extends RecyclerView.ViewHolder {
```

Comment:
It would be good to use more informative class names, so anyone that looks at code can instantly know the purpose of the class